rclcpp/include/rclcpp/qos.hpp

```diff
@@ -153,6 +201,33 @@ class RCLCPP_PUBLIC QoS
   QoS &
   avoid_ros_namespace_conventions(bool avoid_ros_namespace_conventions);
 
+  HistoryPolicy
+  get_history() const;
```

Comment:
Why not:

```c++
HistoryPolicy
history() const;
```

That would follow the style set already by `NodeOptions`, e.g.:

https://github.com/ros2/rclcpp/blob/ba1056fe63590e4253cf362f766fce46be860086/rclcpp/include/rclcpp/node_options.hpp#L100-L114

Also, all the accessors need at least a minimal docstring brief.

Reply:
I avoided that because I felt the difference between `reliable()` being a setter and `reliability()` being a getter is subtle, but it's true it's more consistent with other places.

Reply:
Well, let's not introduce inconsistency. Let's either keep it consistent with what is there or update all of them. Personally I think the existing style is fine, but if others agree with you, I'm fine with changing it.

Reply:
> Well, let's not introduce inconsistency. Let's either keep it consistent with what is there or update all of them. Personally I think the existing style is fine, but if others agree with you, I'm fine with changing it.

Yeah, I agree inconsistency isn't nice, done in https://github.com/ros2/rclcpp/pull/1467/commits/18c71adfcb46e4cc6a6299e59cb1db30b280bbdc.

My preference for the explicit `get_` prefix is probably a non-native speaker issue, my mind has problems to process the difference between the noun and the adjective (reliability/reliable) at first glance.